src/coreclr/src/vm/threads.inl

```diff
@@ -23,11 +23,11 @@
 
 #ifndef DACCESS_COMPILE
 
-#ifndef __GNUC__
-EXTERN_C __declspec(thread) ThreadLocalInfo gCurrentThreadInfo;
-#else // !__GNUC__
+#ifdef _MSC_VER
+__declspec(selectany) __declspec(thread) ThreadLocalInfo gCurrentThreadInfo;
+#else
 EXTERN_C __thread ThreadLocalInfo gCurrentThreadInfo;
```

Comment:
You are probably going to need to change this to `extern` instead of `EXTERN_C` since the symbol will be mangled when defined in `threads.cpp`.

Reply:
It works fine as is - the CI is all green. The definition will inherits the `EXTERN_C` from the declaration.

Reply:
I don’t understand why this is okay actually, but if it builds then I guess it is okay.